python/tank/pipelineconfig.py

```diff
@@ -327,12 +378,27 @@ def _update_metadata(self, updates):
         self._pc_id = curr_settings.get("pc_id")
         self._pc_name = curr_settings.get("pc_name")
 
+    def get_pipeline_config_file_location(self):
```

Comment:
can this be private? why are we adding it to the interface? do we really need a method?